### ISSUE TITLE
Completed qBitorrent torrents not deleted

### ISSUE THREAD

**Issue by aormond** [Sonarr/Sonarr#6655]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Sonarr no longer deletes completed (and paused) torrents from qBittorrent (version 4.6.3) even when the option is enabled.

### Expected Behavior

Paused torrents (that have reached their seeding limits) should be removed from qBittorrent.

### Environment

```markdown
- OS: unraid
- Sonarr: 4.0.2.1183
- Docker Install: lscr.io/linuxserver/sonarr
- Using Reverse Proxy: 
- Browser: 
- Database:
```

### What branch are you running?

Main

### Trace Logs?

[sonarr.trace.txt](https://github.com/Sonarr/Sonarr/files/14738615/sonarr.trace.txt)

### Anything else?

As far as I can tell, qBittorrent may have removed the 'deletePerm' command from the API, which is preventing Sonarr (and Radarr) from removing completed torrents.

**Comment by markus101** [comment]
Please use one of the support channels: [forums](https://forums.sonarr.tv/), [subreddit](https://www.reddit.com/r/sonarr/), [discord ](https://discord.sonarr.tv/), or [IRC](https://web.libera.chat/?channels=#sonarr) for support/questions.

I don't see any removal attempts, do you have `Remove Completed Downloads` enabled? The limits look right for removal, but unless you cut off the logs before it attempted to remove it.

> As far as I can tell, qBittorrent may have removed the 'deletePerm' command from the API

Yes, it was removed with their old API, but v2 API uses the correct command.

**Comment by cooperaj** [comment]
I'm seeing this issue as well. I have definitely enabled the option. It stopped working at some point in the last couple of months and doesn't appear to be a sonarr only thing, it's also affecting radarr and lidarr.

**Comment by aormond** [comment]
I ended up removing all the indexers and re-syncing them from Prowlarr. It turned out that not all the indexers were being properly synced anymore because of something I had misconfigured. They were still showing in Sonarr, but I guess they weren't being updated from Prowlarr anymore? Anyway, I deleted them and re-synced them.

I also changed the seed limits so they are set per indexer in Prowlarr instead of qBittorrent. Now everything is working fine. I'm not sure what the solve actually was. I hope this helps.
